### PR TITLE
Automated policy migration

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- fix https://github.com/RasaHQ/rasa/issues/6820
- add command `rasa data convert config` to migrate `FallbackPolicy`, `MappingPolicy`, `TwoStageFallbackPolicy`
- introduce constants for fallback parameters (affects the fallback policies + `RulePolicy` and `FallbackClassifier`)

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by wochinge** [comment]
docs should also include these changes https://github.com/RasaHQ/rasa/issues/6863

**Comment by wochinge** [comment]
@m-vdb Do you want to merge the documentation in here or directly to `master`? 

@m-vdb @ricwo Apart from the CLI command and its arguments itself, everything has already been reviewed. I'd implement [this comment](https://github.com/RasaHQ/rasa/issues/6820#issuecomment-702246054) in a separate PR.

**Comment by m-vdb** [comment]
re documentation, I'm starting now, but it doesn't matter as long as we do it 👍

**Comment by wochinge** [comment]
Ok thanks 🙌 Then I suggest to give this another review and then merge this asap and do the other things in separate, smaller PRs.

**Review by ricwo — APPROVED** [review]
It would be great if we could test the sys exits with status one for the specific exit reason in each case, other than that it looks great 👍

File: rasa/shared/core/domain.py [hunk 1]
```diff
@@ -461,6 +461,17 @@ def __init__(
         self.store_entities_as_slots = store_entities_as_slots
         self._check_domain_sanity()
 
+    def __deepcopy__(self, memo: Optional[Dict[int, Any]]) -> "Domain":
+        """
+        This dunder method makes it possible to use deepcopy directly
+        on a `Domain` instance.
+
+        See https://docs.python.org/3/library/copy.html#copy.deepcopy
+        for more implementation
```
[the remark this answers is not in the document]
Comment (wochinge) [reply rasa/shared/core/domain.py:470]: For sure! I didn't do a good job when reviewing this 🙈

File: rasa/core/config.py [hunk 2]
```diff
@@ -27,3 +51,168 @@ def load(config_file: Optional[Union[Text, Dict]]) -> List["Policy"]:
         config_data = config_file
 
     return PolicyEnsemble.from_dict(config_data)
+
+
+def migrate_fallback_policies(config: Dict) -> Tuple[Dict, Optional["StoryStep"]]:
```
[the remark this answers is not in the document]
Comment (wochinge) [reply rasa/core/config.py:59]: it's there 👀 `Tuple[Dict, Optional["StoryStep"]]:`

[the remark this answers is not in the document]
Comment (ricwo) [reply rasa/core/config.py:59]: 🤦

**Comment by wochinge** [comment]
Thanks for the quick review @ricwo  🚀 🙌 ❤️

**Pull request merged**